README.md

```diff
@@ -85,6 +85,12 @@ There are a few obvious considerations to ensure we achieve that goal:
 - This API—which allows background "pings" to the refresh endpoint when the user is not directly active—must not enable long-term tracking of a user when they have navigated away from the connected site.
 - Each session has a separate new key created, and it should not be possible to detect that different sessions are from the same device.
 
+### Enterprise support
+
+While DBSC addresses a general problem of session hijacking, and can be applicable to any _browser_ consumer, it is possible to expand this protocol to better support enterprise use cases. By adding specifics to key generation and enabling the switch if there is a specific policy administered on the device, we can provide a more secure environment for enterprise users. This is the goal of DBSC(E), which is an extension to DBSC. The high-level design of DBSC(E) is described in the [DBSC(E) Overview](<DBSC(E)/Overview.md>).
+
+DBSC(E) removes the vulnerability DBSC has, where a malware, if already present in the device during the key generation, can take over the session even if it is eventually bound to the device. By assuring a clean state key generation and introducing device key chaining, DBSC(E) can mitigate this vulnerability: More details about the importance of DBSC are here: [Why DBSC(E)?](<DBSC(E)/Overview.md#why-dbsce>)
```

Comment:
Nit: "where **a** malware", maybe just "where malware"
"By assuring a **clean state** key generation"  Do you want to stick with "clean room" here as it's the verbiage you used in the DBSC(E) doc?

Reply:
Reworked this section with more precise language.